### PR TITLE
Keep the selected items order after export

### PR DESCRIPTION
**Pull request by heitara**

### CODE REVIEW

**Comment by Ludotrico** [comment]
Not sure if you saw my comment on the other thread so:

@heitara thank you for investigating! I just pasted your changes in my project that is currently running the up to date version of the master. Unfortunately the problem persists. Images seem to always be placed before videos, disregarding order.

https://youtu.be/LonRlghD8-w

**Comment by heitara** [comment]
I saw that. I'll test again tomorrow again and will send video if it works on my end.

**Comment by heitara** [comment]
@Ludotrico I've double checked the version in the [branch](https://github.com/heitara/YPImagePicker/tree/feature/issue-488) and it keeps the order of the items. I'm picking image, then video, and so on. I have a screen reccodring that proves it but I can send it directly to you. Could you double check that you are building from the correct repo/branch?

Could you paste the console output from the example app - This is what I get once I finalize the selection:
`YPAssetZoomableView >>> No image
🧀 photo(p: YPImagePicker.YPMediaPhoto)
🧀 video(v: YPImagePicker.YPMediaVideo)
🧀 photo(p: YPImagePicker.YPMediaPhoto)
🧀 video(v: YPImagePicker.YPMediaVideo)
🧀 photo(p: YPImagePicker.YPMediaPhoto)
🧀 video(v: YPImagePicker.YPMediaVideo)
🧀 photo(p: YPImagePicker.YPMediaPhoto)
🧀 video(v: YPImagePicker.YPMediaVideo)
🧀 photo(p: YPImagePicker.YPMediaPhoto)
🧀 video(v: YPImagePicker.YPMediaVideo)
Picker deinited 👍 `

**Comment by Ludotrico** [comment]
@heitara Thanks for the reply, this is in my pod-file for my app:

`pod 'YPImagePicker', :git => 'https://github.com/heitara/YPImagePicker.git' `

Unfortunately, it does not build.

Screen Shot 2020-08-16 at 4 37 12 PM

**Comment by heitara** [comment]
You have to add `, :branch => 'feature/issue-488'` to specify the branch.

**Comment by Ludotrico** [comment]
Great work @heitara ! I can confirm that the order is now preserved. What was the issue?

**Comment by heitara** [comment]
Once all assets are exported those are not sorted again. Like we discussed earlier each asset takes different time to be exported, this is why images were always before the video files. We should merge the fix now. cc  @NikKovIos @s4cha

**Comment by Ludotrico** [comment]
@s4cha  @NikKovIos it has been a week, can you please merge @heitara's branch?

File: Source/Pages/Gallery/YPLibraryVC.swift [hunk 1]
```diff
@@ -510,6 +516,34 @@ public class YPLibraryVC: UIViewController, YPPermissionCheckable {
                 }
                 
                 asyncGroup.notify(queue: .main) {
+                    //TODO: sort the array based on the initial order of the assets in selectedAssets
+                    resultMediaItems.sort {(first, second) -> Bool in
+                        var firstAsset:PHAsset?
```
Comment (NikKovIos) [inline Source/Pages/Gallery/YPLibraryVC.swift]:
```suggestion
                        var firstAsset: PHAsset?
```

File: Source/Pages/Gallery/YPLibraryVC.swift [hunk 2]
```diff
@@ -510,6 +516,34 @@ public class YPLibraryVC: UIViewController, YPPermissionCheckable {
...
+                        var secondAsset:PHAsset?
```
Comment (NikKovIos) [inline Source/Pages/Gallery/YPLibraryVC.swift]:
```suggestion
                        var secondAsset: PHAsset?
```

File: Source/Pages/Gallery/YPLibraryVC.swift [hunk 3]
```diff
@@ -510,6 +516,34 @@ public class YPLibraryVC: UIViewController, YPPermissionCheckable {
                 }
                 
                 asyncGroup.notify(queue: .main) {
+                    //TODO: sort the array based on the initial order of the assets in selectedAssets
+                    resultMediaItems.sort {(first, second) -> Bool in
```
Comment (NikKovIos) [inline Source/Pages/Gallery/YPLibraryVC.swift]:
```suggestion
                    resultMediaItems.sort { (first, second) -> Bool in
```

**Pull request merged**